core/modules/widgets/action-listops.js

```diff
@@ -3,87 +3,89 @@ title: $:/core/modules/widgets/action-listops.js
 type: application/javascript
 module-type: widget
 
-Action widget to apply list operations to any tiddler field (defaults to the 'tags' field of the current tiddler)
+Action widget to apply list operations to any tiddler field (defaults to the 'list' field of the current tiddler)
 
 \*/
 (function () {
 
-  /*jslint node: true, browser: true */
-  /*global $tw: false */
-  "use strict";
+    /*jslint node: true, browser: true */
+    /*global $tw: false */
+    "use strict";
 
-  var Widget = require("$:/core/modules/widgets/widget.js").widget;
+    var Widget = require("$:/core/modules/widgets/widget.js")
+        .widget;
 
-  var ActionListopsWidget = function (parseTreeNode, options) {
-    this.initialise(parseTreeNode, options);
-  };
+    var ActionListopsWidget = function (parseTreeNode, options) {
+        this.initialise(parseTreeNode, options);
```

Comment:
It seems the initial PR was merged with a tab setting 2. So your source with tabs "should" show the diffs you see. Seems to be an oversight.